web/book/src/syntax/expressions-and-operators.md

```diff
@@ -15,21 +15,87 @@ filter circumference > 10 && color != "red"
 
 ## Operator precedence
 
-This table gives formal description of operator precedence. Because function
-calls have the lowest precedence, nested function calls or arguments that start
-or end with an operator require parentheses.
+This table shows operator precedence. Use parentheses `()` to prioritize
+operations and for function calls (see below.) _Question: Are parentheses
+operators? Should they be in this table?_
 
 <!-- markdownlint-disable MD033 — the `|` characters need to be escaped, and surrounded with tags rather than backticks   -->
 
-| Group          | Operators         | Precedence | Associativity |
-| -------------- | ----------------- | ---------- | ------------- |
-| identifier dot | `.`               | 1          |               |
-| unary          | `- + ! ==`        | 2          |               |
-| range          | `..`              | 3          |               |
-| mul            | `* / %`           | 4          | left-to-right |
-| add            | `+ -`             | 5          | left-to-right |
-| compare        | `== != <= >= < >` | 6          | left-to-right |
-| coalesce       | `??`              | 7          | left-to-right |
-| and            | `&&`              | 8          | left-to-right |
-| or             | <code>\|\|</code> | 9          | left-to-right |
-| function call  |                   | 10         |               |
+|          Group | Operators         | Precedence | Associativity |
+| -------------: | ----------------- | :--------: | :-----------: |
+| identifier dot | `.`               |     1      |               |
+|          unary | `- + ! ==`        |     2      |               |
+|          range | `..`              |     3      |               |
+|            mul | `* / %`           |     4      | left-to-right |
+|            add | `+ -`             |     5      | left-to-right |
+|        compare | `== != <= >= < >` |     6      | left-to-right |
+|       coalesce | `??`              |     7      | left-to-right |
+|            and | `&&`              |     8      | left-to-right |
+|             or | <code>\|\|</code> |     9      | left-to-right |
+|  function call |                   |     10     |               |
+|   _parens (?)_ | `()`              |     ??     |   see below   |
+
+## Parentheses
+
+PRQL uses parentheses `()` for several purposes:
+
+- Parentheses group operands to control the order of evaluation, for example:
+  `((1 + x) * y)`
+
+- Parentheses set off an [inner transform](./inner-transforms.md) for the
```

Comment:
Is the "set off" term clear?

Reply:
If you're asking about it, I guess "set off" is not clear :-) And I don't mean like a firecracker...

I've changed it to "denote" to see if that reads better.